pop-api/src/v0/mod.rs

```diff
@@ -18,4 +19,6 @@ pub(crate) enum RuntimeCall {
 	Balances(balances::BalancesCall),
 	#[codec(index = 50)]
 	Nfts(nfts::NftCalls),
+	#[codec(index = 52)]
```

Comment:
Need to feature gate these so they arent all included in compilation.

Reply:
The problem I faced here is that when the runtime has different features enabled then the contract, the encoding and decoding of this enum can vary. 

For example, the contract only has nfts feature, resulting in encoding of the variant `0`. Now if the runtime has all the features enabled the `0` encoded value decodes to a `Balances` call.

Reply:
But that is what the index is supposed to be doing. Seems like maybe the codec index attribute isnt being used?

Reply:
I see, will have another look at it.

Reply:
This was a problem for `RuntimeStateKeys` but solved with the same solution, thanks!